pkg/oauth/apiserver/apiserver.go

```diff
@@ -86,14 +82,6 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 	return s, nil
 }
 
-func (c *completedConfig) V1RESTStorage() (map[string]rest.Storage, error) {
```

Comment:
Since we're getting rid of it (yay \o/), please also remove the same code in the user apiserver

Reply:
done